src/Storage/Collection/LazyCollection.php

```diff
@@ -26,16 +29,50 @@ public function add($element)
     }
 
     /**
-     *  Force loads the proxy objects and returns the real objects
+     * {@inheritdoc}
+     *
+     * Force loads the proxy objects and returns the real objects.
      */
-    public function serialize()
+    public function toArray()
     {
         $output = [];
         foreach ($this as $element) {
+            /** @var Content $proxy */
             $proxy = $element->getProxy();
             $output[] = $proxy->getContenttype() . '/' . $proxy->getSlug();
         }
 
         return $output;
     }
```

Comment:
OK, removed the interface and implementation methods, and changed the name of that to `getProxy()` to match `\Bolt\Storage\EntityProxy` … I'll keep digging as to what's calling what internally.